package-releases/overlays/test/cronjob.yaml

```diff
@@ -3,104 +3,8 @@ kind: CronJob
 apiVersion: batch/v1beta1
 metadata:
   name: package-releases
+  labels:
+    app: thoth
+    component: package-releases
```

Comment:
We don't want to have this as a disabled cronjob in test?

Reply:
I set `suspend: true` for test

Reply:
test is done, so we dont need it to executing in test.
if further testing is needed we can switch it back on.